### PR TITLE
Fixing possibility of adding duplicate entries to list of AsteriskQueueEntryImpl

### PR DESCRIPTION
**Pull request by bbankowski**

### CODE REVIEW

File: src/main/java/org/asteriskjava/live/internal/AsteriskQueueImpl.java [hunk 1]
```diff
@@ -354,27 +354,34 @@ void createNewEntry(AsteriskChannelImpl channel, int reportedPosition, Date date
...
-                serviceLevelTimerTasks.put(qe, timerTask);
-            }
-        }
-
         synchronized (entries)
         {
-            entries.add(qe); // at the end of the list
+			if (getEntry(channel.getName()) != null)
+			{
+				logger.debug("Ignored duplicate queue entry during population in queue " + name + " for channel "
```
Comment (bsutton) [inline src/main/java/org/asteriskjava/live/internal/AsteriskQueueImpl.java:361]: Add a debug level check here.
if (logger.isDebugEnabled())

File: src/main/java/org/asteriskjava/live/internal/AsteriskQueueImpl.java [hunk 2]
```diff
@@ -354,27 +354,34 @@ void createNewEntry(AsteriskChannelImpl channel, int reportedPosition, Date date
...
+
+			entries.add(qe); // at the end of the list
 
             // Keep the lock !
             // This will fire PCE on the newly created queue entry
             // but hopefully this one has no listeners yet
             shift();
         }
 
+		long delay = serviceLevel * 1000L;
```
Comment (bsutton) [inline src/main/java/org/asteriskjava/live/internal/AsteriskQueueImpl.java:374]: What was the reason for moving this code after the queue add?

Comment (rlsutton1) [reply src/main/java/org/asteriskjava/live/internal/AsteriskQueueImpl.java:374]: this makes sense to me, if the entry wasn't added then there is no need to schedule a sevicelevel task

File: src/main/java/org/asteriskjava/live/internal/AsteriskQueueImpl.java (hunk 1 above)
Comment (bbankowski) [reply src/main/java/org/asteriskjava/live/internal/AsteriskQueueImpl.java]: Done.

**Pull request merged**